contrib/exporters/core/extend.go

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
the commit comment is a mess ... please fix

Reply:
This file (core/extend.go) should have been removed. Please ignore code in this file.